gapic-common/lib/gapic/call_options/retry_policy.rb

```diff
@@ -77,14 +77,39 @@ def call error
       def apply_defaults retry_policy
         return unless retry_policy.is_a? Hash
 
-        @retry_codes   ||= retry_policy[:retry_codes]
+        @retry_codes   ||= convert_codes retry_policy[:retry_codes]
         @initial_delay ||= retry_policy[:initial_delay]
         @multiplier    ||= retry_policy[:multiplier]
         @max_delay     ||= retry_policy[:max_delay]
 
         self
       end
 
+      # @private
+      ERROR_CODE_MAPPING = [
+        "OK",
+        "CANCELLED",
+        "UNKNOWN",
+        "INVALID_ARGUMENT",
+        "DEADLINE_EXCEEDED",
+        "NOT_FOUND",
+        "ALREADY_EXISTS",
+        "PERMISSION_DENIED",
+        "RESOURCE_EXHAUSTED",
+        "FAILED_PRECONDITION",
+        "ABORTED",
+        "OUT_OF_RANGE",
+        "UNIMPLEMENTED",
+        "INTERNAL",
+        "UNAVAILABLE",
+        "DATA_LOSS"
```

Comment:
What about `UNAUTHENTICATED` - `16` in this [list](https://grpc.github.io/grpc/core/md_doc_statuscodes.html)?

Reply:
Ah, I didn't see that list. I wouldn't expect `UNAUTHENTICATED` ever to be retryable, but I'll add it for completeness.

Reply:
Want to add a link that list in comments here?

Reply:
Done.

Reply:
And added link.